data/announcements/scheduled.yaml

```diff
@@ -32,7 +32,7 @@ announcements:
 
 - name: Dockershim removal
   startTime: 2022-04-07T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
-  endTime: 2022-05-10T00:00:00 # Expires 2022-05-10, but will change text slightly on release date
+  endTime: 2022-05-09T00:00:00 
```

Comment:
Maybe keep the original endTime since it's already 5/09

Reply:
@reylejano : I am not able to commit this nit (because this line is already resolved when I rebased the PR), is it okay for me to add this nit as a follow up PR after this current PR is merged? Thanks

Reply:
I think you are safe to leave it. In a few hours it will reflex the truth anyway.

Reply:
you don't have to tweak this date/time, just put the banner you want to take precedence first in the list ("if two schedules overlap, the announcement that comes FIRST in the following list takes precedence.")